### PR TITLE
Add <ButtonGroup> Component

### PR DESCRIPTION
**Pull request by prasook-jain**
Button Group is new Component which will take buttons as it's child and group them together.

### CODE REVIEW

**Comment by prasook-jain** [comment]
[ch15416]

File: src/components/Button/ButtonGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  border: none;
+  border-left: 1px solid rgba(255, 255, 255, 0.2);
+`
+
+const ButtonGroup: React.FC<ButtonGroupProps> = props => {
+  const childrenCount = React.Children.count(props.children)
+  return (
+    <ButtonGroupWrapper>
+      {React.Children.map(props.children, (child, i) => {
+        console.log(child)
```
Comment (Kushagra8888) [inline src/components/Button/ButtonGroup.tsx]: remove.

**Comment by arorayash** [comment]
Hey @prasook-jain, any particular reason that you have defined a different knob for each button? Same knob, let's say for selection between primary and secondary button could have been used instead of redefining it for each child in the story. 
Other than that, everything looks good.

**Comment by prasook-jain** [comment]
> Hey @prasook-jain, any particular reason that you have defined a different knob for each button? Same knob, let's say for selection between primary and secondary button could have been used instead of redefining it for each child in the story.
> Other than that, everything looks good.

**Comment by prasook-jain** [comment]
@arorayash did refactoring and removed the extra knobs..

**Pull request merged**